test/wizard.jl

```diff
@@ -86,38 +88,63 @@ for i in available_ports
     break
 end
 
-function readuntil_sift(io::IO, needle)
-    # N.B.: This is a terrible way to do this and works around the fact that our `IOBuffer`
-    # does not block. It works fine here, but do not copy this to other places.
-    needle = codeunits(needle)
-    buffer = zeros(UInt8, length(needle))
-    all_buffer = UInt8[]
-    while isopen(io)
-        new_c = read(io, 1)
-        append!(all_buffer, new_c)
-        if isempty(new_c)
-            # We need to wait for more data, sleep for a bit
-            sleep(0.01)
-            continue
-        end
-
-        buffer = [buffer[2:end]; new_c]
-        if !any(buffer .!= needle)
-            return all_buffer
+@static if true
```

Comment:
Can you elaborate (ideally with a comment in the code) what's going on here?  Reading an `if true` is weird at best, without an explanation.